cni/pkg/nodeagent/server.go

```diff
@@ -69,15 +69,23 @@ func NewServer(ctx context.Context, ready *atomic.Value, pluginSocket string, ar
 		return nil, fmt.Errorf("error initializing kube client: %w", err)
 	}
 
-	cfg := &iptables.IptablesConfig{
+	hostCfg := &iptables.IptablesConfig{
 		RedirectDNS:            args.DNSCapture,
 		EnableIPv6:             args.EnableIPv6,
 		HostProbeSNATAddress:   HostProbeSNATIP,
 		HostProbeV6SNATAddress: HostProbeSNATIPV6,
 	}
 
+	podCfg := &iptables.IptablesConfig{
```

Comment:
Yeah - this is a little messy ATM (because I asked you to turn it off for the host rules) but these can go back to being the same config in a followup.

Reply:
Yes, I agree. We can eventually bring back the old handling once the two configurations converge again